app/views/users/index.html.erb

```diff
@@ -1,8 +1,13 @@
-<h1>Names of all the users </h1>
-
-<% @users.each do |user| %>
-  Name: <%= user.name %>
-  Posts Counter: <%= user.posts_counter %>
-  Posts: <%= user.posts.inspect %>
-  <br>
-<% end %>
\ No newline at end of file
+<h1>Users</h1>
+<ul>
+  <% @users.each do |user| %>
+  <li>
+    <a href="<%= user_path(user) %>">
+      <div class="user-box">
+ <%= render partial: 'pages/user_photo', locals: { user: user } %>
+  <%= render partial: 'pages/user_details', locals: { user: user } %>
+     </div>
+  </a>
+  </li>
+  <% end %>
+</ul>
```

Comment:
| Your Design | Microverse Design |
|-|-|
| ![image](https://github.com/ANTOINE1128/Blog-app/assets/93039359/cd482529-e1ab-4d98-bc11-8be28661ccc0) | ![image](https://github.com/ANTOINE1128/Blog-app/assets/93039359/967108c5-6ea2-4dae-9ef7-589bf5bfec40) |

- [ ] It would be nice if you double-check the original design given by microverse and follow that design while making your views to make your UI professional and attractive 👍
- [ ] Kindly follow the below details 👍

1. Kindly make the image on the left side and size should be similar as you can see in the given design 👍
2. Kindly organize the username and posts_counter on the left as you can see in the design 👀